client/src/store/modules/scheduledEvent/actions.js

```diff
@@ -113,5 +114,31 @@ export default {
                 root: true
             });
         }
+    },
+
+    [actions.UPDATE_EVENT]: async ({ commit, dispatch }, event) => {
+        commit('loader/' + loaderMutations.SET_LOADING, true, { root: true });
+
+        try {
+            const updatedEvent = await scheduledEventService.updateEvent(
+                event.id,
+                eventApiMapper(event)
```

Comment:
I'm not sure if we have to process data on the client side

Reply:
We need to convert data, cause we have startDate on the frontend and we need to send start_date to backend